packages/prosemirror-lwdita-backend/src/api/modules/octokit.module.mts

```diff
@@ -123,24 +122,19 @@ const createFork = async (octokit: Octokit, owner: string, repo: string): Promis
  * @param newBranch - The name of the new branch to be created.
  * @returns A Promise that resolves to a BranchInfo object representing the newly created branch, or undefined if an error occurs.
  */
-const createBranch = async (octokit: Octokit, owner: string, repo: string, newBranch: string): Promise<BranchInfo | undefined> => {
+const createBranch = async (octokit: Octokit, owner: string, repo: string, branch: string, newBranch: string): Promise<BranchInfo | undefined> => {
```

Comment:
tsdoc params need updating.

Reply:
Fixed in b6962288f2f38f863a94da06d3e6f53b77fe42fa .